Migration.lua

```diff
@@ -360,12 +383,58 @@ function Migration:_MigrateCommunity()
     LOG:Message(CLM.L["Import complete"])
 end
 
+function Migration:_MigrateBastion()
```

Comment:
I like this function, and I see good potential in reusing it for CEPGP. Could you make it more generic and then parametrize it for `Bastion`? it seems it's basically changing officer note format for the addons.